docs/concepts/transactions.md

```diff
@@ -0,0 +1,51 @@
+# Transactions 
+
+## Prerequisites
+
+* [Anatomy of an SDK Application](./app-anatomy.md)
+
+## Synopsis
+
+This doc describes how transactions are defined.
+1. [Transactions](#transactions)
+2. [Transaction Definition](#transaction-definition)
+3. [Messages](#messages)
+4. [Handlers and Keepers](#handlers-and-keepers)
+5. [CLI and REST Interfaces](#cli-and-rest-interfaces)
+
+## Transactions
+
+[**Transactions**](https://github.com/cosmos/cosmos-sdk/blob/97d10210beb55ad4bd6722f7186a80bf7cb140e2/types/tx_msg.go#L36-L43)  trigger state changes. They are comprised of metadata and [messages](./modules.md#messages) that instruct modules to perform various actions.
```

Comment:
`Messages` do not instruct modules, they are handled by module's handlers in order to trigger state transitions